pkg/controller/node/node_controller.go

```diff
@@ -440,6 +449,13 @@ func (ctrl *Controller) syncMachineConfigPool(key string) error {
 	if err != nil {
 		return err
 	}
+	// if MachineSelector is empty, get and use NodeSelector
+	if selector == nil {
```

Comment:
this is gonna be always not nil, check https://github.com/openshift/machine-config-operator/pull/625/files#r274715843

Reply:
done